recipes-devtools/python-jsonref/python-jsonref.inc

```diff
@@ -6,6 +6,6 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=a34264f25338d41744dca1abfe4eb18f"
 SRC_URI[md5sum] = "42b518b9ccd6852d1d709749bc96cb70"
 SRC_URI[sha256sum] = "f3c45b121cf6257eafabdc3a8008763aed1cd7da06dbabc59a9e4d2a5e4e6697"
 
-inherit pypi setuptools
```

Comment:
its ok since now inherit setuptools3 is moved to recipe from .inc file.

Reply:
@kraj setuptools where moved from *.inc file to the .bb file, however only for python3 version (setuptools3). If you would like to still build the python2 version of jsonref it will be missing a setuptools. Isn't it correct?

Reply:
Sorry I missed to remove that file, I will remove it.

Reply:
> @kraj setuptools where moved from *.inc file to the .bb file, however only for python3 version (setuptools3). If you would like to still build the python2 version of jsonref it will be missing a setuptools. Isn't it correct?

then you will write another recipe perhaps like python2-jsonref which inherits setuptools 
but I will avoid using py2 at this moment its dead.